tests/api/test_product.py

```diff
@@ -1082,6 +1082,68 @@ def test_product_create_without_category_and_true_is_published_value(
     assert errors[0]["message"] == "You must select a category to be able to publish"
 
 
+def test_product_create_with_collections_webhook(
+    staff_api_client,
+    permission_manage_products,
+    collection,
+    product_type,
+    category,
+    monkeypatch,
+):
+    query = """
+    mutation createProduct($productTypeId: ID!, $collectionId: ID!, $categoryId: ID!) {
+        productCreate(input: {
```

Comment:
There are a lot of mutations: `productCreate` in this file. Maybe we could extract some of them to the single fixture or variable?

Reply:
I think it's beyond the scope of this PR. We'll be working on tests refactoring soon and maybe this will be a good chance to improve it.